package.json

```
@@ -88,7 +88,7 @@
   "peerDependencies": {
     "@libsql/kysely-libsql": ">=0.3.0 <0.5.0",
     "@tediousjs/connection-string": ">=0.5.0 <0.6.0",
-    "better-sqlite3": ">=7.6.2 <8.0.0",
+    "better-sqlite3": ">=9 <12",
```

why < 12? if i may ask?

---

[better-sqlite3](https://github.com/WiseLibs/better-sqlite3/releases/tag/v12.1.1) already released `v12.1.1`.
They seem to be releasing quite often lately; could we accept all until a new major update comes by?

```suggestion
    "better-sqlite3": ">=12.0.0"
```